frontend/src/auth/hooks/useGetUserAccount.tsx

```diff
@@ -53,13 +44,17 @@ export function useGetUserAccount(): UserAccountContextType | undefined {
       }
     }
 
+    if (auth?.isLoading || (auth?.isAuthenticated && !user)) {
```

Comment:
```suggestion
    if (!!auth?.isLoading || (auth?.isAuthenticated && !user)) {
```

ou

```suggestion
    if (auth?.isLoading ?? (auth?.isAuthenticated && !user)) {
```

(pour ESLint)

Reply:
En effet, je prends la 1ère solution !